lib/src/flutter/events/accelerometer/accelerometer_failed_event.dart

```diff
@@ -0,0 +1,12 @@
+part of '../../../../telegram_web_app.dart';
+
+/// **Bot API 8.0+**
+/// 
+/// Occurs if a request to start accelerometer tracking fails.
+/// [eventHandler] receives an object with the single field error, describing the reason for the failure. Possible values for error are:
+/// 
+/// **UNSUPPORTED** – Accelerometer tracking is not supported on this device or platform.
+class AccelerometerFailedEvent extends TelegramEvent {
+  AccelerometerFailedEvent(void Function(String error) eventHandler)
```

Comment:
@OlegNovosad I think you should wrap the error inside an object according to the api definition.

Reply:
According to the api it is just string error, just the same as we do we device orientation and others.